libs/single-spa-angular/internals/src/dom.ts

```diff
@@ -1,19 +1,5 @@
 import { DomElementGetter, BaseSingleSpaAngularOptions } from './types';
 
-export function removeApplicationFromDOMIfIvyEnabled<T extends BaseSingleSpaAngularOptions>(
```

Comment:
This makes this a breaking change, which is okay. We can publish a breaking version for Angular 12